Userland/Applications/Calculator/main.cpp

```diff
@@ -64,6 +64,19 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         widget->set_entry(KeypadValue { Crypto::SignedBigInteger::create_from(27182818284), 10 });
     }));
 
+    auto& round_menu = window->add_menu("&Round");
+    round_menu.add_action(GUI::Action::create("&0", [&](auto& action) {
+        widget->set_rounding_length(action.text().substring(1).to_uint().value());
```

Comment:
uhhh...`0` is less code and less work than `action.text().substring(1).to_uint().value()`